README.md

```diff
@@ -29,6 +29,8 @@
 
     `npm run test:headed`
 
+- create a `.env` file by copying the contents from `.env.sample` and add the respective values you want to use for testing 
+
```

Comment:
QUESTION: Can we provide a sample .env file, e.g. stored in Passbolt, that any developer could drop into their local environment?

Reply:
moving discussion to slack about passbolt